seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -73,9 +73,9 @@
       "title": "Declare JavaScript Variables",
       "description": [
         "When we store data in a <code>data structure</code>, we call it a <code>variable</code>. These variables are no different from the x and y variables you use in math.",
-        "Let's create our first variable and call it \"myName\".",
-        "You'll notice that in <code>myName</code>, we didn't use a space, and that the \"N\" is capitalized. JavaScript variables are written in <code>camel case</code>. An example of camel case is: camelCase.",
-        "Now, use the <code>var</code> keyword to create a variable called <code>myName</code>. Set its value to your name, in double quotes.",
+        "Let's create our first variable and call it \"ourName\".",
+        "You'll notice that in <code>ourName</code>, we didn't use a space, and that the \"N\" is capitalized. JavaScript variables are written in <code>camel case</code>. An example of camel case is: camelCase.",
+        "Now, use the <code>var</code> keyword to create a variable called <code>ourName</code>. Set its value to your name, in double quotes.",
```

Comment:
This and the translated version should also still be `myName`. Thanks for spending the time to correct this.